client/components/App.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Reviews from './Reviews.jsx';
+import TotalReviewScore from './TotalReviewScore.jsx';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      courseNumber: 1,
+      reviews: [],
+      totalReviewScore: {}
+    };
+
+  }
+
+  componentDidMount() {
+    this._isMounted = true;
+
+    fetch('http://localhost:3007/api/userReviews/1')
+      .then(response => response.json())
+      .then(data => {
+        if (this._isMounted) {
```

Comment:
I like the use of _isMounted.  I've had to implement this before when I've gotten errors, I should probably make a habit of it to make my code more 'durable'.

There's also a way to abort a fetch request if the component is unmounted.  If you want to check it out in my SyllabusesService, lines 10 and11 define the abortControllers, 18 and 39 attach them to the fetches, and 78 is what triggers an abort on the GET/api/syllabus.  I seem to have forgotten to add the trigger for images abort.

Reply:
Cool, I'll have a look! Yeah, I was getting error and the _isMounted solution solved the problem. I hadn't used it before either.